store/migration/10001__init_schema.sql

```diff
@@ -836,6 +836,10 @@ CREATE TABLE task (
             "CANCELED"
         )
     ),
+    -- not_before_ts specifies the earliest allowed timing for a task to be executed
+    -- NOTE that the execution timing is also constrained by the allowed window period specified by the env which it belong
```

Comment:
by the env which it belong -> by the env which it belongs or by the belonging environment

Reply:
done